docs/Configuration.md

```diff
@@ -3,6 +3,9 @@
 ## file
 
 ```toml
+# If specified, wait this long for the server to start up.
```

Comment:
don't hesitate to do this into a different PR as it is unrelated

Reply:
Sure thing: #2134